src/covid_shared/cli_tools.py

```diff
@@ -137,14 +151,14 @@ def update_from_file(self, metadata_key: str, metadata_file: typing.TextIO):
         """Loads a metadata file from disk and stores it in the key."""
         self._metadata[metadata_key] = yaml.load(metadata_file)
 
-    def dump(self, metadata_file_path: Union[str, Path]):
+    def dump(self, metadata_file_path: Union[str, Path], secrets: List = None):
```

Comment:
Be more permissive on typing with the public api if it makes sense.  E.g. you could reasonably accept Tuples and Sets here depending on how you decide to implement.